tests/test_cli/test_analytics.py

```diff
@@ -1,132 +1,94 @@
 """Test suite testing command for opting out of analytics."""
-import os
-from typing import Dict, Union
-from unittest import mock
+from unittest.mock import patch
 
-import pytest
-import yaml
 from typer.testing import CliRunner
 
 from matcha_ml.cli.cli import app
-from matcha_ml.services.global_parameters_service import GlobalParameters
 
-INTERNAL_FUNCTION_STUB = "matcha_ml.services.global_parameters_service.GlobalParameters"
+INTERNAL_FUNCTION_STUB = "matcha_ml.core.core"
 
 
-@pytest.fixture(autouse=True)
-def teardown_singleton():
-    """Tears down the singleton before each test case by clearing the current object."""
-    GlobalParameters._instance = None
+def test_cli_analytics_command_help(runner: CliRunner) -> None:
+    """Test cli for analytics command help.
 
-
-@pytest.fixture
-def expected_configuration() -> Dict[str, Union[str, bool]]:
-    """Pytest fixture to return expected configuration.
-
-    Returns:
-        Dict[str, Union[str, bool]]: Dictionary containing expected configuration
+    Args:
+        runner (CliRunner): typer CLI runner
     """
-    return {"analytics_opt_out": False, "user_id": "dummy_user_id"}
+    # Invoke analytics command with help option
+    result = runner.invoke(app, ["analytics", "--help"])
 
+    # Exit code 0 means there was no error
+    assert result.exit_code == 0
 
-def test_opt_out_subcommand(
-    runner: CliRunner,
-    matcha_testing_directory: str,
-    expected_configuration: Dict[str, Union[str, bool]],
-) -> None:
-    """Test opt-out command works.
-
-    Args:
-        runner (CliRuner): Mock runner
-        matcha_testing_directory (str): Temp directory
-        expected_configuration (Dict[str, Union[str, bool]]): Dictionary containing expected configuration
-    """
-    config_file_path = os.path.join(
-        matcha_testing_directory, ".matcha-ml", "config.yaml"
+    # Assert string is present in cli output
+    assert (
+        "Enable or disable the collection of anonymous usage data (enabled by default)."
+        in result.stdout
     )
 
-    expected_configuration["analytics_opt_out"] = True
 
-    # Check if config file is not present
-    assert not os.path.exists(config_file_path)
+def test_cli_analytics_command_default_help(runner: CliRunner) -> None:
+    """Test cli for analytics command help.
```

Comment:
Could this be more descriptive?

Reply:
Absolutely, I'll reword it to 'test_cli_analytics_command_defaults_to_help'.